### ISSUE TITLE
PDF not generating without table border and below 10 rows

### ISSUE THREAD

**Issue by viveksudalai** [dompdf/dompdf#1458]
I am doing project in cakePHP 3 and i used "DaoAndCo/cakephp-dompdf"  for creating PDF.  I have three column table with border "1" in the CTP file. PDF is not generating when the table have only few rows. One more problem is if I change table border to 0 then it's stop generating PDF.

I don't want the border for the columns. so tried to change the TD border color to white. Then i applied
Style border color white to all 3 TD to make it as invisible in the PDF.
If I applied style to 2 and 3 column then again it is stop generating PDF. But it is working when I applied only to first column. I tried with colspan also but dint work.

cakephp function 

```
public function pdftest() { 
	$this->viewBuilder()->layout('empty');
		 $this->set('labellist',$data); 
	        $filename = 'userlabel';
          $this->viewBuilder()
            ->className('Dompdf.Pdf')
            ->layout('Dompdf.label')
			->template('pdf/pdftest')
            ->options(['config' => [
				'orientation' => 'portrait',
				'size' => 'label',
                'filename' => $filename,
                'render' => 'download',
        ]]);     
	}
```

pdftest.ctp
```          
                   <table border="1" cellspacing="1" cellpadding="10" width ="99%">
	           <tr>
		<td>
			 <table width="100%" border="1">
			    <?php
					$count = 20;
			        for($j=0;$j<$count;$j++){
				 echo "<tr>
                                <td style='border: 1px solid #FFFFFF'>Test</td>
			         <td>Test</td>
			         <td>Test</td></tr>";
		        }
			    ?>
			</table>
		</td>
	</tr>
         </table>
 ```

Please help on this issue

**Comment by bsweeney** [comment]
Sorry for the delayed response. If you're still having issues we'll need to see a sample of the HTML used to generate the PDF.

**Comment by viveksudalai** [comment]
Please see above Cakephp function and ctp file

**Comment by bsweeney** [comment]
I'm a little confused by your layout and by your question. Can you rephrase it some.

I did notice one potential issue, however, in your use of a container table. Dompdf is able to handle tables within tables, but there is a limitation with regard to how large an individual cell can be. If any table cell exceeds the height of the page then dompdf will be unable to process it correctly and will enter an infinite loop (see #98).

If all you want is a border around the table but now the table cells you can do the following:

```
<style>
  table { border-collapse: collapse; border: 1px solid black; }
</style>

<table width="100%">
  <tr>
    <td>Test</td>
    <td>Test</td>
    <td>Test</td>
  </tr>
  ...
</table>
```

**Comment by JohanKrab** [comment]
I'm new to dompdf.
I try to display a table with singe borders, but get space between the cells.
I used the border-collapse: collapse style but that doesn't'solve.
When test in a browser it works!
I added the code, and the pdf file.
Also i use the latest version (0.8.5)
What do i miss?
====================================================
code:
<?PHP
defined('_JEXEC') or die('Restricted access');
require_once JPATH_COMPONENT_SITE  . '/helpers/dompdf/autoload.inc.php';
use Dompdf\Dompdf;


$html ='<style type="text/css">';
$html .='/*place your css if any here*/';
$html .='table {';
$html .='  border-collaps: collapse;';
$html .='}';
$html .='table td, th {';
$html .='  border: 1px solid black;';
$html .='}';
$html .='table td img {';
$html .='	width: 20px;';
$html .='	height: auto;';
$html .='}';
$html .='</style>';
$html .='<div>';
$html .='  <table ID="reporttable">';
$html .='  <caption>';
$html .='  <h4>Inventarisatie van beschikbaarheid leden</h4>';
$html .='  <h4>Optreden: ' . $this->items[0]->title . '<h4>';
$html .='  </caption>';


$html .='<thead>';
$html .='      <tr>';
$html .='		<th>Naam</th>';
$html .='		<th></th>';
$html .='		<th>Aangepast</th>';
$html .='      </tr>';
$html .='    </thead>';
$html .='    <tbody>';

	$html .='			<tr>';
	$html .='				<td>John Doe</td>';
	$html .='				<td value=' . $item->available . '>Yes</td>';
	$html .='				<td>12-mei-2020</td>';
	$html .='			</tr>';
	$html .='			<tr>';
	$html .='				<td>Jane Doe</td>';
	$html .='				<td value=' . $item->available . '>No</td>';
	$html .='				<td>10-mei-2020</td>';
	$html .='			</tr>';

$html .='</tbody>';
$html .='  </table>';
$html .='</div>';

$options= new \Dompdf\Options();
$options->set('defaultFont', 'Arial');
$options->set('isRemoteEnabled', true);

$dompdf = new Dompdf($options);
$dompdf->set_option('isHtml5ParserEnabled', true);
$dompdf->setPaper('A4', 'portrait');
$dompdf->setBasePath(JPATH_COMPONENT_SITE . '/helpers/dompdf');
$dompdf->loadHtml($html);
$dompdf->render();
$dompdf->output();

$filename = 'oisreport.pdf';
ob_end_clean();

$dompdf->stream($filename);
?>
[oisreport.pdf](https://github.com/dompdf/dompdf/files/4764764/oisreport.pdf)
